src/OrchardCore/OrchardCore.DisplayManagement.Liquid/OrchardCoreBuilderExtensions.cs

```diff
@@ -68,19 +69,7 @@ public static OrchardCoreBuilder AddLiquidViews(this OrchardCoreBuilder builder)
                 o.MemberAccessStrategy.Register<ShapeMetadata>();
                 o.MemberAccessStrategy.Register<CultureInfo>();
 
-                o.Scope.SetValue("Culture", new ObjectValue(new LiquidCultureAccessor()));
-                o.MemberAccessStrategy.Register<LiquidCultureAccessor, FluidValue>((obj, name, ctx) =>
-                {
-                    return name switch
-                    {
-                        nameof(CultureInfo.Name) => new StringValue(CultureInfo.CurrentUICulture.Name),
-                        "Dir" => new StringValue(CultureInfo.CurrentUICulture.GetLanguageDirection()),
-                        nameof(CultureInfo.NativeName) => new StringValue(CultureInfo.CurrentUICulture.NativeName),
-                        nameof(CultureInfo.DisplayName) => new StringValue(CultureInfo.CurrentUICulture.DisplayName),
-                        nameof(CultureInfo.TwoLetterISOLanguageName) => new StringValue(CultureInfo.CurrentUICulture.TwoLetterISOLanguageName),
-                        _ => NilValue.Instance
-                    };
-                });
+                o.Scope.SetValue("Culture", new CultureValue(CultureInfo.CurrentUICulture));
```

Comment:
I think we can't pass the `CultureInfo.CurrentUICulture` here because this is set to the singleton that is `TemplateOptions`. So if two users have different UI cultures they would use the same one (the one at startup).

So the `CultureInfo.CurrentUICulture` value needs to be used on-demand in the `CultureValue`